aws/src/test/resources/sunstone/aws/cloudformation/eapDomain.yaml

```diff
@@ -0,0 +1,183 @@
+        -   CidrIpv6: "::/0"
+            IpProtocol: "-1"
+      SecurityGroupEgress:
+        -   CidrIp: "0.0.0.0/0"
+            IpProtocol: "-1"
+
+  EC2InstanceEAP:
+    Type: "AWS::EC2::Instance"
+    CreationPolicy:
+      ResourceSignal:
+        Timeout: PT5M
+    DependsOn:
+      - EC2KeyPair
+    Properties:
+      ImageId: !Ref ImageID
+      InstanceType: "t3.medium"
+      KeyName: !Ref EC2KeyPair
+      AvailabilityZone: !GetAtt EC2Subnet.AvailabilityZone
+      Tenancy: "default"
+      SubnetId: !Ref EC2Subnet
+      EbsOptimized: false
+      SecurityGroupIds:
+        - !Ref EC2SecurityGroup
+      SourceDestCheck: true
+      IamInstanceProfile: EC2InstanceRole
+      BlockDeviceMappings:
+        -   DeviceName: "/dev/sda1"
+            Ebs:
+              Encrypted: false
+              VolumeSize: 10
+              VolumeType: "gp2"
+              DeleteOnTermination: true
+      Tags:
+        -   Key: "cf-template"
+            Value: !Ref defaultTag
+        -   Key: "cf-template-resource"
+            Value: !Join [ "-", [!Ref defaultTag, "ec2instance"]]
+        -   Key: "Name"
+            Value: !Ref instanceName
+      HibernationOptions:
+        Configured: false
+      CpuOptions:
+        CoreCount: 1
+        ThreadsPerCore: 2
+      EnclaveOptions:
+        Enabled: false
+      UserData:
+        Fn::Base64:
+          !Sub |
+          #!/usr/bin/env bash
+          JBOSS_HOME=/opt/rh/eap7/root/usr/share/wildfly
+          
+          yum install -y unzip python3-pip
+          pip3 install https://s3.amazonaws.com/cloudformation-examples/aws-cfn-bootstrap-py3-latest.tar.gz
+          
+          INTERNAL_IP_ADDRESS=`ip addr show | grep eth0 -A 2 | head -n 3 | tail -n 1 | awk '{ print $2 }' | sed "s-/24--g" | cut -d'/' -f1`
+          echo $INTERNAL_IP_ADDRESS > /root/internal-ip-address.txt
+          echo "JAVA_OPTS=\"\$JAVA_OPTS \\
+              -Djboss.bind.address=$INTERNAL_IP_ADDRESS \
+              -Djboss.bind.address.private=$INTERNAL_IP_ADDRESS \
+              -Djboss.bind.address.management=$INTERNAL_IP_ADDRESS\"" >> $JBOSS_HOME/bin/domain.conf
+          echo 'HOST_CONTROLLER_JAVA_OPTS="$HOST_CONTROLLER_JAVA_OPTS $JAVA_OPTS"' >> $JBOSS_HOME/bin/domain.conf
+          
+          echo "" >> $JBOSS_HOME/bin/domain.conf
+          
+          # set up users (for remote management connections and application connections)
+          $JBOSS_HOME/bin/add-user.sh -u admin -p pass.1234 -r ManagementRealm -g SuperUser -e
+          
+          # set up env vars
+          echo "export JAVA_HOME=/usr/lib/jvm/java" >> /root/.bashrc
+          echo "export JBOSS_HOME=$JBOSS_HOME" >> /root/.bashrc
+          echo "export JAVA_HOME=/usr/lib/jvm/java" >> /home/ec2-user/.bashrc
+          echo "export JBOSS_HOME=$JBOSS_HOME" >> /home/ec2-user/.bashrc
+          
+          # start EAP
+          systemctl enable eap7-domain
+          systemctl restart eap7-domain
+          sleep 20
```

Comment:
I have had issues on standalone with 20 sec only and increased it to 30. Domain start takes even longer. Can you set it to 60? IK it is rather long, so can you also add periodic check for status?